include/mitsuba/core/distr_1d_cond.h

```diff
@@ -0,0 +1,1225 @@
+#pragma once
+
+#include <drjit/dynamic.h>
+#include <drjit/tensor.h>
+#include <mitsuba/core/logger.h>
+#include <mitsuba/core/math.h>
+#include <mitsuba/core/vector.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
```

Comment:
I'm on the fence about class naming. I kind of expected `IrregularConditionalDistribution1D` but that's starting to get long.